packages/utilities/src/GlobalSettings.ts

```diff
@@ -7,28 +7,13 @@
  * callers to get called back when it mutates.
  */
 
+// tslint:disable:no-any
+
 const GLOBAL_SETTINGS_PROP_NAME = '__globalSettings__';
 const CALLBACK_STATE_PROP_NAME = '__callbacks__';
 
-// tslint:disable-next-line:no-any
-let _global: { [key: string]: any };
 let _counter = 0;
 
-if (typeof window !== 'undefined') {
-  _global = window;
-} else if (typeof global !== 'undefined') {
-  _global = global;
-} else {
-  _global = {};
-}
-
-// tslint:disable-next-line:no-any
-let _globalSettings: { [key: string]: any } = (_global[GLOBAL_SETTINGS_PROP_NAME] = _global[GLOBAL_SETTINGS_PROP_NAME] || {
-  [CALLBACK_STATE_PROP_NAME]: {}
-});
-
-const _callbacks = _globalSettings[CALLBACK_STATE_PROP_NAME];
-
 /**
  * Change description used for change callbacks in GlobalSettings.
```

Comment:
These changes were an attempt to remove the word `global`, which webpack ends up bailing on tree shaking for. I thought it would fix it to just move this logic into a helper function, but it didn't. Still think it's a reasonable change to make.